chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/AddedToRoomTest.kt

```diff
@@ -0,0 +1,103 @@
+package com.pusher.chatkit.rooms.state
+
+import assertk.assertThat
+import assertk.assertions.containsOnly
+import assertk.assertions.isNotNull
+import com.pusher.chatkit.state.ChatkitState
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+
+class AddedToRoomTest : Spek({
+
+    describe("Given an initial state of empty rooms") {
+        val joinedRoomsState = JoinedRoomsState(
+                hashMapOf<String, JoinedRoomInternalType>(),
+                hashMapOf<String, Int>()
+        )
+        val currentState = ChatkitState(joinedRoomsState = joinedRoomsState)
+
+        describe("when a new added to room event is received") {
+            val addedToRoomRoom = AddedToRoom(
+                    JoinedRoomsStateTestUtil.roomOne,
+                    1
```

Comment:
named param would be good for unread count here

Reply:
👍